templates/accounts_test.go

```diff
@@ -0,0 +1,53 @@
+/*
+ * Flow Go SDK
+ *
+ * Copyright 2019-2020 Dapper Labs, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package templates_test
+
+import (
+	"github.com/onflow/flow-go-sdk"
+	"github.com/onflow/flow-go-sdk/templates"
+	"github.com/stretchr/testify/require"
+	"testing"
+)
+
+func TestCreateAccount(t *testing.T) {
+	// Converting transaction arguments to Cadence values can increase their size.
+	// If this is not taken into account the transaction can quickly grow over the maximum transaction size limit.
+	t.Run("Transaction should not grow uncontrollably in size", func(t *testing.T) {
+		contractLen := 1000
+		contractCode := make([]byte, contractLen)
```

Comment:
Just to double check -- does the zeroed byte array translate to a non-empty hex string? e.g. `"000000..."` instead of just `""` or `"0"`

Reply:
yes: https://play.golang.org/p/-BlkWurnsrv

I could make them random, but I am not comfortable with random elements in tests (not that it would matter here).

Reply:
@janezpodhostnik Nope that's okay! I just wasn't sure what the behaviour was and wanted to check.